From 0c5054210b3e309c90ee35e2064b27acc913315a Mon Sep 17 00:00:00 2001
Message-ID: <0c5054210b3e309c90ee35e2064b27acc913315a.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 22:24:15 +0000
Subject: [PATCH v3 50/66] gnu:
 go-github-com-aws-aws-sdk-go-v2-service-ssooidc: Move to (gnu packages
 golang-web).

* gnu/packages/golang.scm (go-github-com-aws-aws-sdk-go-v2-service-ssooidc): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: I7c1fc41bb4b7ae8b92204e3daee5628602328384
---
 gnu/packages/golang-web.scm | 11 +++++++++++
 gnu/packages/golang.scm     | 10 ----------
 2 files changed, 11 insertions(+), 10 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 5dd73bd4db..4bc9c944cb 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -192,6 +192,17 @@ (define-public go-github-com-aws-aws-sdk-go-v2-service-sso
     (propagated-inputs
      (list go-github-com-aws-smithy-go))))
 
+(define-public go-github-com-aws-aws-sdk-go-v2-service-ssooidc
+  (package
+    (inherit go-github-com-aws-aws-sdk-go-v2)
+    (name "go-github-com-aws-aws-sdk-go-v2-service-ssooidc")
+    (version "1.13.10")
+    (arguments
+     '(#:import-path "github.com/aws/aws-sdk-go-v2/service/ssooidc"
+       #:unpack-path "github.com/aws/aws-sdk-go-v2"))
+    (propagated-inputs
+     (list go-github-com-aws-smithy-go))))
+
 (define-public go-github-com-aws-smithy-go
   (package
     (name "go-github-com-aws-smithy-go")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 32756ad0ce..0ef1ba4edf 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -11300,16 +11300,6 @@ (define-public go-github-com-dvsekhvalnov-jose2go
     (home-page "https://github.com/dvsekhvalnov/jose2go")
     (license license:expat)))
 
-(define-public go-github-com-aws-aws-sdk-go-v2-service-ssooidc
-  (package
-    (inherit go-github-com-aws-aws-sdk-go-v2)
-    (name "go-github-com-aws-aws-sdk-go-v2-service-ssooidc")
-    (version "1.13.10")
-    (arguments
-     '(#:import-path "github.com/aws/aws-sdk-go-v2/service/ssooidc"
-       #:unpack-path "github.com/aws/aws-sdk-go-v2"))
-    (propagated-inputs (list go-github-com-aws-smithy-go))))
-
 (define-public go-github-com-aws-aws-sdk-go-v2-service-sts
   (package
     (inherit go-github-com-aws-aws-sdk-go-v2)
-- 
2.41.0

